index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>HTML-Forms</title>
+</head>
+<body>
+    <header>
+      <nav class="nav-text">
+              Already have an Intuit account?
+              <p><a href="#">I forgot my User ID or Password</a></p>
+          <button class="sign-in-button"><i class="fa fa-lock" aria-hidden="true"></i>Sign In</button>
+      </nav>
+    </header>
+
+    <section class="form-title">
+        <img alt="topicons" class="logoicons" id="logo" src="images/intuitlogo.png" />
+        <img alt="topicons" class="logoicons" id="logoicons" src="images/topicons.png" />
```

Comment:
[REQUIRED]
Instead of using only one image for the 3 logos(mint, quickbooks, turbotax), use the 3 logos(you can find them in the original page) with text for the names of the products.